File: casadaApi/src/main/java/com/casada/casadaApi/entities/City.java
package com.casada.casadaApi.entities;

import lombok.Data;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.io.Serializable;

/**
 * $table.getTableComment()
 */
@Data
@Entity
@Table(name = "city")
public class City implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    @Column(name = "citypostalcode", nullable = false)
    private Long citypostalcode;

    @Column(name = "cityname", nullable = false)
    private String cityname;

}

File: casadaGUI/casada/lib/data/city.dart
class City {
  int? citypostalcode;
  String? cityname;

  City({this.citypostalcode, this.cityname});

  City.fromJson(Map<String, dynamic> json) {
    citypostalcode = json['citypostalcode'];
    cityname = json['cityname'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['citypostalcode'] = this.citypostalcode;
    data['cityname'] = this.cityname;
    return data;
  }
}